lib/blockscore/actions/retrieve.rb

```diff
@@ -11,7 +11,7 @@ module Actions
     # => #<BlockScore::Person:0x007fe39c424410>
     module Retrieve
       module ClassMethods
-        RESOURCE_ID_FORMAT = /\A[a-f0-9]+\z/.freeze
+        RESOURCE_ID_FORMAT = /\A[a-f0-9]+\z/
```

Comment:
I think you still want to freeze regexps, as without `.freeze` the eigenclass could be mutated.